tarun_kumar_PostgreAssignment/src/services/mark-service.ts

```diff
@@ -0,0 +1,30 @@
+import { dbQuery } from "../db/db";
+
+
+export async function addMark(student_id: string, subject_id: string, marks: number) {
+    try {
+        const query = 'insert into mark (student_id, subject_id, marks) values ($1, $2, $3)';
```

Comment:
can you check https://sequelize.org/master/ package, this is ORM. It provides methods to do CRUD you can pass object around.

Reply:
you dont have to do string interpolation for every operation.